### ISSUE TITLE
Implement scalable event logging

### ISSUE THREAD

**Issue by jennlove-um** [tl-its-umich-edu/my-learning-analytics#969]
## Describe your problem or feature you'd like added
Currently, Currently when requests come in for loading a view we fetch data from DB, event log into a DB table and send the response to UI. We need to rethink how this needs to be designed to fit the scale. 

One approach: serve the page/data up to the user then separately log the event

Use a message queue and send events to that.  This would be dependent on whether we use AWS or Google.

**Comment by jennlove-um** [comment]
Related (future) issue: #310 instrument MyLA with caliper

**Comment by jonespm** [comment]
On the [pinax-eventlog](https://github.com/pinax/pinax-eventlog/#overview) page they mention.

For small sites, it should be good enough to use inline but you might want to consider wrapping calls to the log() method and queue them in a job manager like celery or pyres so that the calls become asynchronous.

**Comment by pushyamig** [comment]
Good Idea, probably for the next release. I wanted the asynchronous logging for better performance. We might want to upgrade the version of Pinax to latest as that now supports Django 3 and MySQL

**Comment by jennlove-um** [comment]
Additional load testing should be done to understand at what point this becomes an issue for scale.
